src/components/BonusNumberInput.js

```diff
@@ -0,0 +1,25 @@
+import customCreateElement from "../utils/customElement.js";
+
+export default class BonusNumberInput {
+  constructor($target, bonusNumber) {
+    this.render($target, bonusNumber);
+  }
+
+  render($target, bonusNumber) {
+    const $div = customCreateElement({
+      tagName: "div",
+      className: "lotto-number-input-wrap bonus-input-wrap",
+    });
+
+    const $text = customCreateElement({ tagName: "span", text: "보너스 번호" });
+    const $input = customCreateElement({
+      tagName: "input",
+      className: "number-input",
+    });
+    $input.value = bonusNumber === 0 ? "" : bonusNumber;
+
+    $div.appendChild($text);
+    $div.appendChild($input);
+    $target.appendChild($div);
+  }
+}
```

Comment:
이렇게도 가능할 것 같은데요~ 클래스로 작성해주신 이유가 궁금합니다~!

```suggestion
export default function renderBonusNumberInput($target, bonusNumber) {
  const $div = customCreateElement({
    tagName: "div",
    className: "lotto-number-input-wrap bonus-input-wrap",
  });

  const $text = customCreateElement({ tagName: "span", text: "보너스 번호" });
  const $input = customCreateElement({
    tagName: "input",
    className: "number-input",
  });
  $input.value = bonusNumber === 0 ? "" : bonusNumber;

  $div.appendChild($text);
  $div.appendChild($input);
  $target.appendChild($div);
}
```

Reply:
처음에 컴포넌트의 상태를 클래스 속성처럼 관리하기 위해 클래스를 사용했습니다.
상태 변경은 setState와 같은 메서드를 통해서만 이루어지도록 하고
다른 메서드에서는 공통적으로 해당 속성을 참조할 수 있도록 클래스 구조를 선택했습니다. 그래서 전체적으로 컴포넌트는 다 클래스 구조를 갖도록 했습니다!

Reply:
예전에 setState가 있었던 레거시라고 이해하면 될까요? 
거의 모든 클래스에 setState가 없으니, 함수가 좀 더 어울린다고 생각했어요~!

Reply:
넵! 그리고 render를 따로 해주는 부분이 있어서 엘리먼트 생성부분과 render부분을 나눌 수 있도록 클래스를 사용한 이유도 있습니다!
통일성도 좋지만 이런 단순한 부분은 어쩌면 함수가 더 어울리는게 아닌가 하는 고민도 드는 거 같습니다